### ISSUE TITLE
previousRun and currentRun return null all the time

### ISSUE THREAD

**Issue by Earu** [Hexagon/croner#226]
**Describe the bug**
previousRun and currentRun return null all the time.

**To Reproduce**
```ts
let date = Cron("0 0 11 * *", { timezone: "Europe/London" }).previousRun();
console.log("PREVIOUS", date);
```

**Expected behavior**
The actual date of when the previous run would have ran.

**System:**
 - OS: Windows
 - Runtime: Node.js 
 - Version: v18.15.0

**Comment by Hexagon** [comment]
Hi!

That's actually by design, both previousRun() and currentRun() only return the time of an actual trigger. There is currently not way to list previous matches, but there is an discussion started at https://github.com/Hexagon/croner/discussions/219

**Comment by Hexagon** [comment]
You should be able to do something like

```ts
const job = new Cron("0 0 11 * * *"); // 11:00:00 every day
const today = new Date().setHours(0,0,0,0); // (hours, minutes, seconds, milliseconds)
const firstRunToday = job.nextRun(today); // get the first runTime today
const earlierTriggerToday = firstRunToday && firstRunToday < new Date();
const laterTriggerToday = job.currentRun().setHours(0,0,0,0) == today; // currentRun is the same as previousRun but is set before starting the trigger, hence better for this
console.log(earlierTriggerToday, laterTriggerToday);
```

**Comment by Earu** [comment]
Thanks ! That's more or less what I was trying to achieve, do you thinks maybe a similar function could make its way into a future version ? I'm sure I'm not the only one having a use case for checking whether a cron expr is gonna run/has run on a specific date (in my case today).

**Comment by Hexagon** [comment]
Yep, I'll consider adding a feature like this, just got to figure out how to make is as usable as possible 👍
